packages/contracts/test/test-helpers/data/index.ts

```diff
@@ -1,5 +1,38 @@
 import { create2Tests } from './create2.test.json'
 import { rlpTests } from './rlp.test.json'
+import * as fs from 'fs'
+
+const createSynthetixJSON = () => {
+  const files = {}
+  const sanitizeLibs = (str: string): string => {
+    return str
+      .split('__$')
+      .join('000')
+      .split('$__')
+      .join('000')
+  }
+  const dir = __dirname + '/synthetix/optimized/'
```

Comment:
This will only run on unix like systems, you should do something like:
`path.join(__dirname, 'synthetix', 'optimized');`

https://nodejs.org/api/path.html#path_path_join_paths